app/src/lib/editors/win32.ts

```diff
@@ -10,663 +10,595 @@ import {
+      publisher === 'Microsoft Corporation',
+  },
+  {
+    name: 'Visual Studio Code (Insiders)',
+    registryKeys: [
+      // 64-bit version of VSCode (user) - provided by default in 64-bit Windows
+      {
+        key: HKEY.HKEY_CURRENT_USER,
+        subKey:
+          'SOFTWARE\\Microsoft\\Windows\\CurrentVersion\\Uninstall\\{217B4C08-948D-4276-BFBB-BEE930AE5A2C}_is1',
+      },
+      // 32-bit version of VSCode (user)
+      {
+        key: HKEY.HKEY_CURRENT_USER,
+        subKey:
+          'SOFTWARE\\Microsoft\\Windows\\CurrentVersion\\Uninstall\\{26F4A15E-E392-4887-8C09-7BC55712FD5B}_is1',
+      },
+      // 64-bit version of VSCode (system) - was default before user scope installation
+      {
+        key: HKEY.HKEY_LOCAL_MACHINE,
+        subKey:
+          'SOFTWARE\\Microsoft\\Windows\\CurrentVersion\\Uninstall\\{1287CAD5-7C8D-410D-88B9-0D1EE4A83FF2}_is1',
+      },
+      // 32-bit version of VSCode (system)
+      {
+        key: HKEY.HKEY_LOCAL_MACHINE,
+        subKey:
+          'SOFTWARE\\Wow6432Node\\Microsoft\\Windows\\CurrentVersion\\Uninstall\\{C26E74D1-022E-4238-8B9D-1E7564A36CC9}_is1',
+      },
+    ],
+    executableShimPath: ['bin', 'code-insiders.cmd'],
+    usesShell: true,
+    expectedInstallationChecker: (displayName, publisher) =>
+      displayName.startsWith('Microsoft Visual Studio Code Insiders') &&
+      publisher === 'Microsoft Corporation',
+  },
+  {
+    name: 'Visual Studio Codium',
+    registryKeys: [
+      // 64-bit version of VSCodium (user)
+      {
+        key: HKEY.HKEY_CURRENT_USER,
+        subKey:
+          'SOFTWARE\\Microsoft\\Windows\\CurrentVersion\\Uninstall\\{2E1F05D1-C245-4562-81EE-28188DB6FD17}_is1',
+      },
+      // 32-bit version of VSCodium (user)
+      {
+        key: HKEY.HKEY_CURRENT_USER,
+        subKey:
+          'SOFTWARE\\Microsoft\\Windows\\CurrentVersion\\Uninstall\\{C6065F05-9603-4FC4-8101-B9781A25D88E}}_is1',
+      },
+      // 64-bit version of VSCodium (system)
+      {
+        key: HKEY.HKEY_LOCAL_MACHINE,
+        subKey:
+          'SOFTWARE\\Microsoft\\Windows\\CurrentVersion\\Uninstall\\{D77B7E06-80BA-4137-BCF4-654B95CCEBC5}_is1',
+      },
+      // 32-bit version of VSCodium (system)
+      {
+        key: HKEY.HKEY_LOCAL_MACHINE,
+        subKey:
+          'SOFTWARE\\WOW6432Node\\Microsoft\\Windows\\CurrentVersion\\Uninstall\\{E34003BB-9E10-4501-8C11-BE3FAA83F23F}_is1',
+      },
+    ],
+    executableShimPath: ['bin', 'codium.cmd'],
+    usesShell: true,
+    expectedInstallationChecker: (displayName, publisher) =>
+      displayName.startsWith('VSCodium') &&
+      publisher === 'Microsoft Corporation',
+  },
+  {
+    name: 'Sublime Text',
+    registryKeys: [
+      {
+        key: HKEY.HKEY_LOCAL_MACHINE,
+        subKey:
+          'SOFTWARE\\Microsoft\\Windows\\CurrentVersion\\Uninstall\\Sublime Text 3_is1',
+      },
+    ],
+    executableShimPath: ['subl.exe'],
+    appInformationExtractor: keys => {
```

Comment:
If we change the `expectedInstallationChecker` for this entry to check whether displayName starts with `Sublime Text` we should be able to remove this `appInformationExtractor` method

Reply:
Nice catch!